### ISSUE TITLE
elasticsearch auth with password

### ISSUE THREAD

**Issue by Happyfeet01** [domainaware/parsedmarc#215]
Is it possible to connect parsedmarc with elasticsearch when used username and password for the index?

Like thi: http://user:passwd@127.0.0.1:9200
 
ES, Kibana and Parsedmarc are installed from Ubuntu repository.

**Comment by seanthegeek** [comment]
Yes — current versions support this two ways: `hosts = https://user:secret@127.0.0.1:9200` (URL-encode special characters), or the separate `user` / `password` (or `api_key`) options in the `[elasticsearch]` section. See https://domainaware.github.io/parsedmarc/usage.html. Closing as implemented.

---
🤖 Triaged and closed by [Claude Code](https://claude.com/claude-code) at the maintainer's request.
